models/ed/R/met2model.ED2.R

```diff
@@ -196,12 +191,10 @@ met2model.ED2 <- function(in.path, in.prefix, outfolder, start_date, end_date, l
       print("Skipping to next year")
       next
     }
-    
-    
+
+
     ## calculate potential radiation in order to estimate diffuse/direct
-    f                <- pi/180 * (279.5 + 0.9856 * doy)
-    et               <- (-104.7 * sin(f) + 596.2 * sin(2 * f) + 4.3 * sin(4 * f) - 429.3 * 
-                           cos(f) - 2 * cos(2 * f) + 19.3 * cos(3 * f)) / 3600  # equation of time -> eccentricity and obliquity
+    et <- PEcAn.data.atmosphere::eccentricity_obliquity(doy)
```

Comment:
I think it's great to encapsulate this, but you could probably go even further and encapsulate the whole solar geometry calculation of solar zenith angle as well (i.e. everything up to cosz below)

Reply:
Yeah, I was thinking that too. I went through it quickly and pulled this out because it was really obvious and I saw it in a few places. But yeah, I'll go ahead and do that, so wait to merge this until I do.